PaddleCV/object_detection/ppdet/utils/visualizer.py

```diff
@@ -103,5 +98,7 @@ def draw_bbox(image, catid2name, bboxes, threshold, is_bbox_normalized=False):
             fill='red')
         if image.mode == 'RGB':
             draw.text((xmin, ymin), catid2name[catid], (255, 255, 0))
+        logger.info("\t {:15s} at {:25} score: {:.5f}".format(catid2name[catid],
```

Comment:
IMHO these logging are not really necessary, at least shouldn't be part of the visualize function

Reply:
change to debug